src/main/kotlin/org/rust/lang/core/psi/RustCompositeElement.kt

```diff
@@ -10,3 +13,21 @@ interface RustCompositeElement : PsiElement {
 
 val RustCompositeElement.containingMod: RustMod?
     get() = PsiTreeUtil.getStubOrPsiParentOfType(this, RustMod::class.java)
+
+/**
+ * Returns the level of the given lint for this element.
+ */
+fun PsiElement.lintLevel(lint: RustLint): RustLintLevel {
```

Comment:
I think it's better to swap arguments: `RustLint.lintLevel(element: PsiElement)`. `PsiElement`s are used everywhere, and you rarely have a need for `lintLevel`.